internal/store/kvbacked.go

```diff
@@ -185,6 +185,9 @@ func NewKVStore(cfg Config) (KVStore, error) {
 		config := etcdclientv3.Config{
 			Endpoints: addrs,
 			TLS:       tlsConfig,
+			DialTimeout: time.Second,
```

Comment:
DialTimeout is just related to creating a connection to an endpoint (the it's kept open), making it of just 1 second is probably too much (k8s uses 20 seconds) and I'm not sure which issue will fix.

Reply:
I think this timeout not matter as keepalive